### ISSUE TITLE
Latest version (0.6.0) of Flutter Fimber is broken

### ISSUE THREAD

**Issue by TNorbury** [magillus/flutter-fimber#97]
Error code:
```
.../Pub/Cache/hosted/pub.dartlang.org/flutter_fimber-0.6.0/lib/flutter_fimber.dart:1:8: Error: Error when reading '.../Pub/Cache/hosted/pub.dartlang.org/fimber-0.6.1/lib/colorize.dart': The system cannot find the file specified.

import 'package:fimber/colorize.dart';
       ^


FAILURE: Build failed with an exception.
```

**Comment by magillus** [comment]
oh, how did my tests run without issues, when I was testing it.

Thank you for reporting, sorry for trouble

**Comment by magillus** [comment]
I fixed it with 0.6.1 - also `fimber_io` has same issue.

**Comment by TNorbury** [comment]
Thanks for the quick fix!
